cpp/tests/reductions/reduction_tests.cpp

```diff
@@ -435,10 +469,11 @@ TYPED_TEST(MultiStepReductionTest, DISABLED_var_std)
   std::vector<T> v = convert_values<T>(int_values);
   cudf::test::fixed_width_column_wrapper<T> col(v.begin(), v.end());
 
-  double var   = calc_var(v, v.size());
-  double std   = std::sqrt(var);
-  auto var_agg = cudf::make_variance_aggregation(/*ddof =*/1);
-  auto std_agg = cudf::make_std_aggregation(/*ddof =*/1);
+  double var      = calc_var(v, v.size());
+  double std      = std::sqrt(var);
+  auto const ddof = 1;
```

Comment:
I'd suggest refactoring `calc_var` above so that it accepts `ddof` as an argument, rather than hard-coding it in the body of that lambda and again on this line.

Reply:
I appreciate that, I should have considered that it would expand the scope. It's always fine to push comments like this to a separate PR or decline them to keep the PR moving. Sorry!